sequencer/src/bin/nasty-client.rs

```diff
@@ -528,7 +528,7 @@ impl<T: Queryable> ResourceManager<T> {
                     // be sure the endpoint is healed.
                     tracing::info!("succeeded after {i} retries; checking health");
                     for _ in 0..self.cfg.min_retries {
-                        f().into_future().await.context(
+                        TryFutureExt::into_future(f()).await.context(
```

Comment:
This an similar changes are because there are 3 traits that implement `into_future` in scope now.